pkg/aks-engine/helpers/azureconst.go

```diff
@@ -0,0 +1,1698 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package helpers
+
+// AUTOGENERATED FILE
+
+// GetAzureLocations provides all azure regions in prod.
+// Related powershell to refresh this list:
+//   Get-AzureRmLocation | Select-Object -Property Location
+func GetAzureLocations() []string {
```

Comment:
where do we use this? 

I feel we don't need this azureconst.go at all.

Reply:
The only thing left in azureconst.go is GetAzureLocations(), which is currently used pkg/aks-engine/engine/output.go and pkg/agent/datamodel/types.go. Maybe we don't need those code that uses it, too? I'll keep it for now and do another pass in the future. There are definitely more code than this that we don't need.